index.bs

```diff
@@ -3060,7 +3061,7 @@ a [=moment=] |triggerTime|, and a [=boolean=] |fenced|:
     1. Set |triggerContextID| to |value|["<code>[=trigger-registration JSON key/trigger_context_id=]</code>"].
 1. Let |attributionScopes| be null.
 1. If |value|["<code>[=trigger-registration JSON key/attribution_scopes=]</code>"] [=map/exists=]:
```

Comment:
Now that we have a separate function for parsing scopes for triggers, can we move this presence check to that algorithm?